.vscodeignore

```diff
@@ -0,0 +1,10 @@
+# Ignore all files by default.
```

Comment:
I found some examples and practices using `.vscodeignore` for white listing: so that we can invoke the `vsce` without changing the working directory or copying the included resources, and the layout for the resources should be the same no matter we minified or not.